spec/Section 3 -- Type System.md

```diff
@@ -232,6 +232,163 @@ Schema extensions have the potential to be invalid if incorrectly defined.
+this syntax covers.
+
+All examples below will assume the following schema:
+
+```graphql example
+directive @private(scope: String!) on FIELD
+
+scalar DateTime
+
+input ReviewInput {
+  content: String
+  author: String
+  businessId: String
+}
+
+interface Address {
+    city: String
+}
+
+type User implements Address {
+    name: String
+    reviewCount: Int
+    friends: [User]
+    email: String @private(scope: "loggedIn")
+    city: String
+}
+
+type Business implements Address {
+    name: String
+    address: String
+    rating: Int
+    city: String
+    reviews: [Review]
+    createdAt: DateTime
+}
+
+type Review {
+    content: String
+    author: User
+    business: Business
+    createdAt: DateTime
+}
+
+union Entity = User | Business | Review
+
+enum SearchFilter {
+    OPEN_NOW
+    DELIVERS_TAKEOUT
+    VEGETARIAN_MENU
+}
+
+type Query {
+    searchBusiness(name: String!, filter: SearchFilter): Business
+}
+
+type Mutation {
+    addReview(input: ReviewInput!): Review
+}
+```
+
+The following table shows examples of Schema Coordinates for elements in the
+schema above:
+
+| Schema Coordinate              | Description                                                         |
+| ------------------------------ | ------------------------------------------------------------------- |
+| `Business`                     | `Business` type                                                     |
+| `User.name`                    | `name` field on the `User` type                                     |
+| `Query.searchBusiness(name:)`  | `name` argument on the `searchBusiness` field on the `Query` type   |
+| `SearchFilter`                 | `SearchFilter` enum                                                 |
+| `SearchFilter.OPEN_NOW`        | `OPEN_NOW` value of the`SearchFilter` enum                          |
+| `@private`                     | `@private` directive definition                                     |
+| `@private(scope:)`             | `scope` argument on the `@private` directive definition             |
+| `Address`                      | `Address` interface                                                 |
+| `Address.city`                 | `city` field on the `Address` interface                             |
+| `ReviewInput`                  | `ReviewInput` input object type                                     |
+| `ReviewInput.author`           | `author` input field on the `ReviewInput` input object type         |
+| `Entity`                       | `Entity` union definition                                           |
+| `DateTime`                     | Custom `DateTime` scalar type                                       |
+| `String`                       | Built-in `String` scalar type                                       |
+
+Schema Coordinates are always unique. Each type, field, argument, enum value, or
+directive may be referenced by exactly one possible Schema Coordinate.
```

Comment:
feels like something worth highlighting and being clear about as a guarantee in the spec